src/py42/clients/securitydata.py

```diff
@@ -468,3 +469,18 @@ def destination_guid(self):
     def node_guid(self):
         """The GUID of the storage node containing the archive."""
         return self._node_guid
+
+
+def _escape_quote_chars_in_token(token):
+    """
+    The `nextPgToken` returned in Forensic Search requests with > 10k results is the eventId
```

Comment:
Does adding events to cases work ok without having to escape the quotes or should we do the same there?

Reply:
oh, that's a good thought. I'll check.

Reply:
Looks like these events break the cases API when the eventId is in the URL.. gonna need to file a bug.